Software/bsw/stub/stub.c

```diff
@@ -72,6 +72,10 @@ void CanIfTxPduUserTriggerTransmit(PduIdType TxPduId, PduInfoType* PduInfoPtr)
 
 }
 
+void CanIfTxPduUserTriggerTransmitName(PduIdType TxPduId, PduInfoType* PduInfoPtr)
```

Comment:
i believe that this function is duplicated in line # 70 with name CanIfTxPduUserTriggerTransmit
so it doesn't matter if its having word "Name" in it's suffix or not but what i believe that matters is having the call back functions here as stub for the <User_TriggerTransmit>, <User_TxConfirmation>, <User_RxIndication> functions that we need in our implementation
like: PduR_CanIfTriggerTransmit() , PduR_CanIfTxConfirmation().... and so on

Reply:
Yeah these formulas are repeated and both are used in CanIf_PbCfg.c , so it left me with linking error , so I just added the definition , Until you edit them in CanIf_PbCfg.c